flume-ng-channels/flume-kafka-channel/src/test/java/org/apache/flume/channel/kafka/TestKafkaChannel.java

```diff
@@ -860,4 +861,85 @@ public static void deleteTopic(String topicName) {
     AdminUtils.deleteTopic(zkUtils, topicName);
   }
 
+  @Test
+  public void testAutomaticOffsetsMigration() throws Exception {
+    String group="testMigrateOffsets-kafka";
```

Comment:
Hi @umesh9794,
Here is an example for violation: missing spaces around '='. I don't know why Trevis is different to github in aspect of the reported line numbers (after applying your patch seem to me that Trevis was correct). You can check style locally by executing `mvn clean install -DskipTests`